packages/dm-core/src/components/ViewCreator/ViewCreator.tsx

```diff
@@ -50,19 +51,25 @@ export const ViewCreator = (props: TViewCreator): JSX.Element => {
       .attributeGet({
         reference: reference,
       })
-      .then((response: any) => {
+      .then((response: AxiosResponse) => {
         setAttribute(response.data)
       })
       .catch((error) => setError(error))
       .finally(() => setIsLoading(false))
   }, [])
 
   if (isLoading) return <Loading />
-  if (error) throw error
+  if (error)
```

Comment:
I think the proper way to do this would be to make an ErrorBoundary plugin that would catch all errors and display them in a nice way. That way we can be sure to avoid the page crashing in an unexpected way

Reply:
adding a note about this to https://github.com/orgs/equinor/projects/189/views/3?filterQuery=is%3Aissue+-cardtype%3AEpic+error+bo&pane=issue&itemId=31823204 :)